bitnami/dataplatform-bp1/Chart.yaml

```
@@ -19,20 +19,25 @@ dependencies:
     name: zookeeper
     repository: https://charts.bitnami.com/bitnami
     version: 6.x.x
+  - condition: wavefront.enabled
+    name: wavefront
+    repository: https://charts.bitnami.com/bitnami
```

It is needed to specify here the major branch for the subcharts
```suggestion
  - condition: wavefront.enabled
    name: wavefront
    repository: https://charts.bitnami.com/bitnami
    version: 1.x.x
```